includes/builder/class-boldgrid-editor-builder.php

```diff
@@ -134,6 +134,14 @@ public static function get_theme_buttons() {
 		global $boldgrid_theme_framework;
 
 		if ( empty( $boldgrid_theme_framework ) ) {
+			/**
+			 * Allow 3rd-Party Themes to add buttons
+			 *
+			 * @param array button_classes {
+			 * 	An key value array of button names and their classes
```

Comment:
Looks like we'll need a bit more spacing: https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#1-1-parameters-that-are-arrays